mysql/datadog_checks/mysql/data/conf.yaml.example

```diff
@@ -73,178 +79,210 @@ instances:
     #
-    # use_global_custom_queries: true
+    # use_global_custom_queries: 'true'
 
-    ## @param custom_queries - list - optional
-    ## Each query must have 2 fields:
+    ## @param custom_queries - list of mappings - optional
+    ## A list of custom queries used to retrieve custom metrics.
+    ##
+    ## The following fields are expected:
     ##
-    ## 1. query - The SQL to execute. It can be a simple statement or a multi-line script.
-    ## 2. columns - The list representing each column, ordered sequentially from left to right.
-    ##              The number of columns must equal the number of columns returned in the query.
-    ##              There are 2 required pieces of data:
-    ##                a. name - The suffix to append to `clickhouse.` to form
-    ##                          the full metric name. If `type` is `tag`, this column is
-    ##                          considered a tag and applied to every
-    ##                          metric collected by this particular query.
-    ##                b. type - The submission method (gauge, monotonic_count, etc.).
-    ##                          Use `tag` to tag each metric in the row
-    ##                          with the name and value of the item in this column.
-    ##                          Use the `count` type to perform aggregation for queries that
-    ##                          return multiple rows with the same or no tags.
-    ## 3. tags (optional) - A list of tags to apply to each metric.
+    ## query:
+    ##   The SQL query to execute. It can be a simple statement, or a multi-line script.
+    ## columns:
+    ##   A list representing each column, ordered sequentially from left to right.
+    ##   The number of columns must be equal to the number of columns returned in the `query`.
+    ##
+    ##   Each column is an object with the following fields:
+    ##
+    ##   name:
+    ##     The suffix to append to `mysql.` to form the full metric name.
+    ##     If `type` is `tag`, this column is considered a tag and applied to every metric
+    ##     collected by this query.
+    ##   type:
+    ##     The submission method (gauge, monotonic_count, etc.).
+    ##     Use the 'tag' type to tag each metric in the row with the name and value
+    ##     of the item in this column.
+    ##     Use the 'count' type to perform aggregation for queries that return multiple
+    ##     rows with the same or no tags.
+    ## tags: (optional)
+    ##   A list of tags to apply to each metric.
     #
     # custom_queries:
-    #   - query: |  # Use the pipe if you require a multi-line script.
-    #       SELECT foo,
-    #              COUNT(*)
-    #       FROM table.events
-    #       GROUP BY foo
+    #   - query: SELECT foo, COUNT(*) FROM table.events GROUP BY foo
     #     columns:
-    #       # Columns without a name are ignored. Add additional columns to skip:
-    #       # - {}
```

Comment:
Not sure I understood this comment: is it that eg if the query returns 3 columns, we can do `- {}` on the first one and define the 2 others to only submit metrics for the last 2 columns? Not sure this is correct, since if the user wants to "skip columns" then they can just omit them in the `SELECT` clause 🤔 So I removed it for now.

Reply:
@ofek added this feature recently so he must remember what he meant